clang-tools-extra/docs/clang-tidy/checks/llvmlibc/inline-function-decl.rst

```diff
@@ -3,6 +3,7 @@
 llvmlibc-inline-function-decl
 =============================
 
-Checks that all implicit and explicit inline functions in header files are
-tagged with the ``LIBC_INLINE`` macro. See the `libc style guide
-<https://libc.llvm.org/dev/code_style.html>`_ for more information about this macro.
+Checks that all implicitly and explicitly inline functions in header files are
+tagged with the ``LIBC_INLINE`` macro, except for functions implicit to classes.
+See the `libc style guide<https://libc.llvm.org/dev/code_style.html>`_ for more
```

Comment:
There is some documentation error on this line, maybe some space is missing before <

Reply:
I've added the space, hopefully it is happy now.